### PR TITLE
Add new Accordion designs

### PR DESCRIPTION
**Pull request by poteirard**
### 🤔 Questions for the reviewer

1. Do you guys are happy with the icons in CSS?
2. Do I have to remove the Arrow icon as it's not longer used?

### 🖌 Designs

Designs are not defined as 100% as I don't have a reference for the open state. Ask the design team is they are happy. I used this reference: https://marvelapp.com/e14bhc4/screen/67764982/handoff

SHOPFRONT_landing_page_refresh__Q2_2020_

Mine looks like this:

React_components

### CODE REVIEW

**Comment by moshie** [comment]
Do you guys are happy with the icons in CSS?
I see no issue in fact I would say it's better because you don't have extra elements to load. 👍

Do I have to remove the Arrow icon as it's not longer used?
Yes please.

**Comment by lluia** [comment]
@moshie I need to open a PR removing the existing icons and using **font-awesome** ones instead ( and add the `font-awesome` import to **Styleguidist** ), is the last thing I plan to do on **ZRC** before my furlough 📐

File: src/components/organisms/Accordion/AccordionHeader/AccordionHeader.tsx [hunk 1]
```diff
@@ -14,46 +14,48 @@ export interface IAccordionHeader extends HTMLAttributes<HTMLButtonElement> {
...
+const Cross = styled.span<{ active: boolean }>`
+  width: 10px;
+  height: 10px;
+  // This background creates the ➖
+  background: linear-gradient(to bottom, transparent 35%, ${colors.grey} 35%, ${colors.grey} 65%, transparent 65%)
+    ${({ active }) =>
+      !active &&
+      // This adds the vertical stick to create ✚
+      `, linear-gradient(to right, transparent 35%, ${colors.grey} 35%, ${colors.grey} 65%, transparent 65%)`};
+`;
```
Comment (lluia) [inline src/components/organisms/Accordion/AccordionHeader/AccordionHeader.tsx:47]: Have you tested this looks good in all browser? 💭 

Ha ha ha maybe we can replace them with a **font-awesome** SVG later when we refactor the icons 👍🏻

**Comment by moshie** [comment]
> @moshie I need to open a PR removing the existing icons and using **font-awesome** ones instead ( and add the `font-awesome` import to **Styleguidist** ), is the last thing I plan to do on **ZRC** before my furlough 📐

No problem, the Arrow can still be removed

File: src/components/organisms/Accordion/AccordionHeader/AccordionHeader.tsx (hunk 1 above)
Comment (poteirard) [reply src/components/organisms/Accordion/AccordionHeader/AccordionHeader.tsx:47]: Firefox, Chrome and Safari

**Comment by poteirard** [comment]
I will leave the deprecated button to @lluia PR as he's going to kill all the folder

File: src/components/organisms/Accordion/AccordionHeader/AccordionHeader.tsx (hunk 1 above)
Comment (lluia) [reply src/components/organisms/Accordion/AccordionHeader/AccordionHeader.tsx:47]: If it really doesn't pose a layout problem lets leave it like this then 📐

**Pull request merged**